src/components/SuchUI.js

```diff
@@ -0,0 +1,14 @@
+export default function SearchUI() {
+    return `
+      <div>
+        <input id="search-input" type="text" placeholder="Nach Notizen suchen..." />
+        <button onclick="searchNotes()">Suchen</button>
+      </div>
+    `;
+  }
+  
+  window.searchNotes = function () {
+    const query = document.getElementById('search-input').value;
+    console.log('Suchanfrage:', query);
+  };
```

Comment:
please don't use functions attached to the window object as it can cause memory leaks. Please use normal react methods instead as outlined [here](https://react.dev/learn/referencing-values-with-refs#example-building-a-stopwatch)